AOJ/06/06_B.py

```diff
@@ -0,0 +1,30 @@
+n = int(input())
+group = []
+while n != 0:
+    a,b1 = input().split()
+    b = int(b1)
+    group += [(a,b)]
+    n -= 1
+
+scount = list(range(1,14))
+hcount = list(range(1,14))
+ccount = list(range(1,14))
+dcount = list(range(1,14))
+for (picture,figure) in group:
+    if picture == 'S':
+        scount.remove(figure)
+    elif picture == 'H':
+        hcount.remove(figure)
+    elif picture == 'C':
+        ccount.remove(figure)
+    elif picture == 'D':
+        dcount.remove(figure)
```

Comment:
上記で辞書型にした場合、条件文がいらなくなります。
- `picure` の値を場合分けしている箇所を、辞書の`key` で参照するようにすると良いと思います。
```suggestion
for picture, figure in group:
    cards[picuture].remove(int(figure))
```